team-arrival-checklist/form.tex

```diff
@@ -0,0 +1,42 @@
+\documentclass[a4paper, 12pt]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage[default,osfigures]{opensans}
+
+\usepackage[margin=0.5in]{geometry}
+\usepackage{tabularx}
+\setlength{\parindent}{0cm}
+
+\newcolumntype{Y}{>{\centering\arraybackslash}X}
+
+\begin{document}
+
+\begin{center}
+
+    {\huge{Team Arrival Checklist}}
+
+    \medskip
+
+\end{center}
+
+When a team arrives, please check them off on this sheet by filling in their time of arrival.
+\bigskip
+
+{\large{Date}: \hrulefill}
```

Comment:
Just listing other things we might want to capture here. Feel free to ignore if you've already considered these:
- the volunteer on reception
- which reception desk it was

I'm not actually sure how useful these would be, but they're pretty easy to capture and might be handy. Super low priority to add though and I'm happy to omit them entirely.

Reply:
I think it's ok to leave these out, i'm not sure how useful they'll be. If we do need to chase something up, the schedule can tell us who to poke.